protocol-spec/specific.md

```diff
@@ -4,13 +4,15 @@
 
 **Description:** Causes a toy that supports Kiiroo style commands to run whatever event may be related. More information on Kiiroo commands can be found in STPIHKAL.
 
+**Introduced In Version:** 0
+
 **Message Version:** 0
 
 **Fields:**
 
 * _Id_ \(unsigned int\): Message Id
 * _DeviceIndex_ \(unsigned int\): Index of device
-* _Command_ \(unsigned int\): Unsigned integer in range \[0-4\].
```

Comment:
Why did this move from uint to string? (I vaguely remember having a conversation about this but can't seem to find it, so mostly looking for a refresher here)

Reply:
This was due to the Kiiroo implementation not matching the schema when 0.1.0 was released